fs/littlefs/lfs_vfs.c

```diff
@@ -1362,6 +1387,11 @@ static int littlefs_bind(FAR struct inode *driver, FAR const void *data,
         }
     }
 
+  if (data && strstr(data, "ro"))
```

Comment:
using strstr this way doesn't seem like a great idea, especially for a very short string as "ro", which can be easily a part of unrelated words.
ideally we should have a parser which recognizes comma separations.

for now, as i guess it's rare for users to want to specify "ro" with other options (forceformat/autoformat), maybe it's good enough to use `!strcmp(data, "ro")`.
what do you think?